flask_app/forms/experiment_chart_form.py

```diff
@@ -0,0 +1,166 @@
+import re
+
+import plotly.express as px
+import numpy as np
+
+from flask_app.lib.hack import get_chart_data
+from flask_app.models.experiment_df_wrapper import ExperimentDfWrapper
+
+PLOTLY_TEMPLATE = 'plotly_white'
```

Comment:
Streamlit seems to have their own custom theme, which is Apache 2.0-licensed, so it should be fine to reuse, but I don't think it matters much, and we could tweak it later ourselves.